src/imports/loggerUtils/ErrorsTransport.js

```diff
@@ -10,14 +10,15 @@ export default class ErrorsTransport extends Transport {
   static factory = options => new ErrorsTransport(options, Config, Sentry)
 
   constructor(opts, Config, Sentry) {
-    const { env, sentryDSN, version, network, remoteLoggingAllowed } = Config
+    const { env, sentryDSN, version, network, remoteLoggingAllowed, release } = Config
 
     super(opts)
 
     if (remoteLoggingAllowed && sentryDSN) {
       Sentry.init({
         dsn: sentryDSN,
-        environment: env
+        environment: env,
+        release
```

Comment:
@yaroslav-fedyshyn-nordwhale 

release version is managed by the ci scripts. it's auto increased each release to qa/prod and stored in package.json (it's just an version of GoodDapp package). So we shouldn't set additional variable for it. We have to read it from package.json and add it to the env dynamically via  webpack's define plugin (pls see https://webpack.js.org/plugins/define-plugin/). The other stuff without changes - it would be env var accessible via process.env, then it passes to the config and in other code places we're using config to read the release version